apps/condo/domains/meter/utils/serverSchema/index.js

```diff
@@ -33,11 +35,12 @@ const getAvailableResidentMeters = async (context, userId) => {
         const residentPropertyId = get(resident, ['property', 'id'])
```

Comment:
find don't have nested structures...it stores only ids, like SQL table, so 
```suggestion
        const residentPropertyId = get(resident, ['property'])
```